app/scripts/collections/app.js

```diff
@@ -0,0 +1,18 @@
+import AppConfig from 'scripts/config';
+import Session from 'scripts/facades/session';
+
+export default class AppCollection extends Backbone.Collection {
+  constructor(...args) {
+    this.url = `${AppConfig.apiPath}/${_.result(this, 'url')}`;
+
+    super(...args);
+  }
+
+  sync(method, model, options) {
+    if (Session.isLoggedIn()) {
+      options.data = _.extend({}, options.data, Session.token);
```

Comment:
Yeah, thx. In the future commits I will try to replace underscore methods with ES6 if it's possible